driver/src/main.rs

```diff
@@ -82,10 +82,6 @@ struct Options {
     #[structopt(long, env = "PRICE_FEED_INFORMATION", default_value = "{}")]
     backup_token_data: TokenData,
 
-    /// Number of seconds the solver should maximally use for the optimization process
-    #[structopt(long, env = "SOLVER_TIME_LIMIT", default_value = "150")]
```

Comment:
I think we can keep this as the target time when the solver returns (3 minutes) and then have the code compute how many seconds are left for that.

Reply:
I'm not sure I understand what you mean. Shouldn't we just always compute how much time the solver has and give it more when we have more?